package.json

```diff
@@ -44,7 +44,7 @@
   },
   "devDependencies": {
     "@monaco-editor/react": "^4.4.5",
-    "@transcend-io/airgap.js-types": "^9.6.0",
+    "@transcend-io/airgap.js-types": "^10.0.0-alpha.2",
```

Comment:
will bump to 10.0.0 after https://github.com/transcend-io/airgap.js-types/pull/82 merges and publishes